src/views/Home.vue

```diff
@@ -1,14 +1,162 @@
+                specifieke operationele informatie beveiligd volgens de
+                eisen die het Rijk hieraan stelt. Alle kaartlagen in het
+                LIWO kunnen vanuit de geo-server rechtstreeks gedeeld
+                worden met systemen van derden, zoals bijvoorbeeld met
+                het Landelijk Crisis Management Systeem (LCMS) en
+                webapplicaties zoals Overstroomik.nl. U kunt de
+                handleiding voor toegang tot de geo-server opvragen bij
+                functioneel beheer LIWO.
+              </p>
+              <h3>Is LIWO een rekenmodel?</h3>
+              <p>
+                Het LIWO is geen rekenmodel. In het landschap van
+                systemen en kennis m.b.t.  overstromingen valt er
+                onderscheid te maken
+                tussen:</p>
+              <ol>
+                <li>Model-/rekeninstrumentaria</li>
+                <li>Gedeelde databases</li>
+                <li>Viewers</li>
+                <li>Crisiscommunicatiesystemen</li>
+                <li>Risicocommunicatietool</li>
+              </ol>
+              <h3>Kennis leverende projecten </h3>
+              <p>
+                Het LIWO is een gedeelde database (2), een viewer (3) en
+                een crisiscommunicatiesysteem (4). Het LIWO bevat
+                informatie die onder andere is gemaakt met
+                model-/rekeninstrumentaria (1) afkomstig van diverse
+                kennisleverende projecten.  Het LIWO levert informatie
+                aan diverse risicocommunicatietools (5).</p>
+              <h3>Kan ik zelf aan de slag met het LIWO?</h3>
+              <p>
+                Daar is het LIWO voor bedoeld. In algemene zin is het
+                voor eenieder die zich niet als waterexpert beschouwt
+                aan te bevelen om voor het toepassen van dit type kennis
+                een expert van het desbetreffende waterschap of
+                Rijkswaterstaat te betrekken. Daarbij kan het voor de
+                gebruiker van belang zijn om zoveel mogelijk aan te
+                sluiten bij informatie die voor Nederland eenduidig en
+                uniform is vervaardigd, zoals de resultaten van het
+                project Veiligheid Nederland in Kaart en het
+                Deltaprogramma; Deze zijn beschikbaar in het LIWO. Voor
+                enkele specifieke toepassingen kan het raadzaam zijn om
+                een ander type informatie te gebruiken.  Bespreek de
+                mogelijkheden met het
+                waterschap.
+              </p>
+            </div>
+          </transition>
+        </div>
+      </div>
+      <p>&nbsp;</p>
+    </section>
   </div>
 </template>
 
 <script>
-// @ is an alias to /src
 export default {
   name: 'home',
-  components: {
+  data () {
+    return {
+      showFaq: false
+    }
   }
 }
 </script>
+<style>
+.site-main, .site-secondary {
+   max-width: none;
+}
+.landing-cover {
```

Comment:
the landing-cover is not responsive

Reply:
Now using consistent landing-page layout.